addons/im_livechat/static/src/core/web/thread_icon_patch.js

```diff
@@ -0,0 +1,13 @@
+import { ThreadIcon } from "@mail/core/common/thread_icon";
+import { _t } from "@web/core/l10n/translation";
+
+import { patch } from "@web/core/utils/patch";
+
+patch(ThreadIcon.prototype, {
+    get defaultChatIcon() {
+        if (this.props.thread.channel_type === "livechat") {
+            return { class: "fa fa-comments", title: _t("Livechat") };
+        }
+        return super.defaultChatIcon;
+    },
```

Comment:
Better than the xpath. In the future we might want to just keep the same default icon as I don't think it adds much value.